<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleConeSearchForm</class>
 <widget class="QWidget" name="SimpleConeSearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cone Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupCone">
     <property name="title">
      <string>Cone Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Radius (deg):</string>
        </property>
        <property name="buddy">
         <cstring>textRadius</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>RA (deg):</string>
        </property>
        <property name="buddy">
         <cstring>textRA</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="textURL"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="textDec"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="textRadius"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dec (deg):</string>
        </property>
        <property name="buddy">
         <cstring>textDec</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Verbosity:</string>
        </property>
        <property name="buddy">
         <cstring>comboVerbosity</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="textRA"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cone URL:</string>
        </property>
        <property name="buddy">
         <cstring>textURL</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboVerbosity">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>1 (minimum)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 (default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 (maximum)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTable">
     <property name="title">
      <string>Table</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="table"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textURL</tabstop>
  <tabstop>textRA</tabstop>
  <tabstop>textDec</tabstop>
  <tabstop>textRadius</tabstop>
  <tabstop>comboVerbosity</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
